utils/vscode/carbon.code-snippets

```diff
@@ -0,0 +1,55 @@
+{
+  "package": {
+    "scope": "carbon",
+    "prefix": "package",
+    "body": "package $fileNameWithoutExt",
+    "description": "Carbon package"
+  },
+  "Main Run function": {
+    "scope": "carbon",
+    "prefix": "Run",
+    "body": ["fn Run() -> i32 {", "\t$1", "\treturn 0;", "}"]
+  },
+  "New Function": {
+    "scope": "carbon",
+    "prefix": "fn",
+    "body": ["fn $1($2) ->  $3 {", "\t$4", "}"]
+  },
+  "If statement": {
+    "scope": "carbon",
+    "prefix": "if",
+    "body": ["if ($1) {", "\t$2", "}"]
+  },
+  "Else if statement": {
+    "scope": "carbon",
+    "prefix": "elif",
+    "body": ["else if ($1) {", "\t$2", "}"]
+  },
+  "For each loop with for": {
+    "scope": "carbon",
+    "prefix": "foreach",
+    "body": ["for (var $1: $2 in $3) {", "\t$4", "}"]
```

Comment:
This probably shouldn't have `var`. It isn't necessary unless you want to mutate the variable in the loop body, which probably isn't the common case.

Reply:
Should I use constant instead? I couldn't actually find constant keyword
So I used it